codebases/signalen.html

```diff
@@ -0,0 +1,136 @@
+      <a href="#img1">
+        <img class="lb-thumb" src="../photos/codebases/signalen/signalen_thumb_g01.png" alt="Signalen product thumbnail">
+      </a>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img1">
+        <a href="#img3" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img2"><img src="../photos/codebases/signalen/signalen_g01.png" alt="Signalen product example 1"></a>
+        <a href="#img2" class="lb-btn lb-btn-next"></a>
+      </div>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img2">
+        <a href="#img1" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img3"><img src="../photos/codebases/signalen/signalen_g02.png" alt="Signalen product example 2"></a>
+        <a href="#img3" class="lb-btn lb-btn-next"></a>
+      </div>
+      <!-- lightbox container hidden with CSS -->
+      <div class="lightbox" id="img3">
+        <a href="#img2" class="lb-btn lb-btn-prev"></a>
+        <a href="#" class="lb-btn lb-btn-close"></a>
+        <a href="#img1"><img src="../photos/codebases/signalen/signalen_g03.png" alt="Signalen product example 3"></a>
+        <a href="#img1" class="lb-btn lb-btn-next"></a>
+      </div>
+    </div>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We're helping Amsterdam and the VNG define a governance structure, including shared feature roadmaps and technical steering.</p>
+    <p>We're also helping make the codebase more conducive to collaboration, including:</p>
+    <ul>
+      <li>Clear branching and gitflow models</li>
+      <li>Automated unit tests and integration testing</li>
+      <li>Engineering guidelines</li>
+      <li>Help prepare the community to empower market parties to offer the solution as SaaS</li>
+    </ul>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Technology</h3>
+    <dl>
+      <dt>Languages and frameworks:</dt>
+      <dd>Django Python</dd>
+    </dl>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Community</h3>
+    <dl>
+      <dt>Created by:</dt>
+      <dd>Amsterdam, the Netherlands (2018)</dd>
+    </dl>
+    <dl>
+      <dt>Replicating:</dt>
+      <dd>Strong interest from other Dutch cities and the VNG (Vereniging van Nederlandse Gemeenten, representing 355 municipalities), as well as Ghent (Belgium) as part of the European SCORE project.</dd>
+    </dl>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Opportunity</h3>
+    <dl>
+      <dt>Scaling potential:</dt>
+      <dd>Other Dutch municipalities</dd>
+    </dl>
+    <p>The Association of Dutch Municipalities (VNG) wants to make it 'one click deployable' from their Common Ground appstore.</p>
```

Comment:
On line 104 we have the long Dutch form. Should we put them all in one place and then just refer to VNG everywhere else?